jbpm-wb-integration/jbpm-wb-integration-client/src/main/java/org/jbpm/workbench/wi/client/editors/deployment/descriptor/items/NamedObjectItemView.java

```diff
@@ -73,6 +78,11 @@ public void init(final NamedObjectItemPresenter presenter) {
     public void onRemoveButtonClicked(final ClickEvent ignore) {
         presenter.remove();
     }
+    
+    @EventHandler("edit-button")
+    public void onEditButtonClicked(final ClickEvent ignore) {
+        presenter.openEditModal(name.textContent, value.textContent);;
```

Comment:
No need to pass values back to the presenter.

Reply:
ok